scss/adminLessonNav.module.scss

```diff
@@ -0,0 +1,98 @@
+@use './_variables.module';
+@use 'sass:color';
+
+.lessons_tabsNav {
+  font-family: 'PT Mono';
+  width: 100%;
+  padding: 18px 23px;
+
+  background: white;
+  border: 1px solid #ededed;
+  box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.07);
+  border-radius: 10px;
+  align-items: center;
+  display: flex;
+  column-gap: 20px;
+}
+
+.lessons__tabsNav__nav {
+  display: flex;
+  column-gap: 20px;
+}
+
+.lessons_tabsNav__nav__item {
+  padding: 7px 16px 7px 16px;
+  border-radius: 4px !important;
+  background-color: variables.$primary !important;
```

Comment:
Used `!important` to override bootstrap styles. I'm aware it's bad practice but it was necessary.

Reply:
Oh, it's possible without it. react-bootstrap components has this `bsPrefix` that reset the component's styles and apply the styles you assign to it.